src/app_service/service/message/service.nim

```diff
@@ -563,10 +563,10 @@ QtObject:
     except Exception as e:
       error "error: ", procName="pinUnpinMessage", errName = e.name, errDesription = e.msg
 
-  proc fetchMessageByMessageId*(self: Service, chatId: string, messageId: string):
+  proc getMessageByMessageId*(self: Service, chatId: string, messageId: string):
```

Comment:
Since this interacts with status-go's database it should be called async. Have you tried to search and scroll to very old message? Is there UI freeze?

Reply:
Hmmm. I though getting a single by id a fast operation. 🤔 
Looks quite smooth to me, even scrolling to an old message.

https://github.com/status-im/status-desktop/assets/25482501/f6d2c97a-fd1d-40a1-a5ee-55009b744286

cc @jrainville @osmaczko

Reply:
> Hmmm. I though getting a single by id a fast operation. 🤔 Looks quite smooth to me, even scrolling to an old message.

Right. Even fast db operations should be done async due to db locking. Long story short, the fast db operation can end up waiting in a queue of db requests, effectively blocking our UI thread. It all depends on how busy the db is at the time of the request.

Reply:
Added `asyncGetMessageById` 👌